packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -335,14 +456,28 @@ $-chevron-expand-more: map-merge(
 }
 
 // Big footer styles
-
 .usa-footer--big {
```

Comment:
Noting that the alignment issue is fixed in new (top) vs old (bottom). 

Social navigation is now right-aligned and matches the signup input above it. Before, social nav had extra space on the right.

![image](https://github.com/uswds/uswds/assets/3385219/66e207b6-8f64-4568-82d0-826496f766eb)